### PR TITLE
Set --version option

### PR DESCRIPTION
**Pull request by szkl**
Signed-off-by: Sonmez Kartal <sonmez@koding.com>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -80,7 +80,7 @@ jobs:
 
       - run:
           name: configure
-          command: ./configure --config dev --host dev.koding.com:8090 --hostname dev.koding.com --countlyApiPort "80"
+          command: ./configure --config dev --version $(git rev-parse --short HEAD) --host dev.koding.com:8090 --hostname dev.koding.com --countlyApiPort "80"
```
Comment (cihangir) [inline .circleci/config.yml]: what is the diff from [version = require('git-rev-sync').short()](https://github.com/koding/koding/blob/master/configure#L152) ?

Comment (szkl) [inline .circleci/config.yml]: [require('git-rev-sync').short](https://github.com/kurttheviking/git-rev-sync-js/blob/master/index.js#L117-L119) just gets the first 8 characters.

[git rev-parse --short](https://github.com/git/git/blob/master/builtin/rev-parse.c#L161) [finds the unique abbreviation](https://github.com/git/git/blob/master/sha1_name.c#L466-L509) of the revision. I think the former is okay for development purposes only.

Comment (cihangir) [inline .circleci/config.yml]: 👍

**Review by cihangir — APPROVED** [review]

**Comment by cihangir** [comment]
Please rebase

**Pull request merged**